core/src/main/java/com/novoda/merlin/MerlinsBeard.java

```diff
@@ -69,4 +69,12 @@ public boolean isConnectedToMobileNetwork() {
         return networkInfo != null && networkInfo.isConnected();
     }
 
+    public String getNetworkInfoSubtypeName() {
+        NetworkInfo networkInfo = getNetworkInfo();
+        if (networkInfo == null || !networkInfo.isConnected()) {
```

Comment:
when connected to WiFi, `networkInfo` is not null and it _is_ connected, but the subtype for WiFi is `""`